src/ui/templates/index.njk

```diff
@@ -16,17 +16,17 @@
     <link rel="apple-touch-icon" href="/assets/images/govuk-apple-touch-icon.png">
 
     <link rel="stylesheet" href="/assets/css/styles.css" />
-    <!--[if IE 8]>
-      <link rel="stylesheet" href="/govuk-frontend-ie8-3.5.0.min.css">
-    <![endif]-->
 
+    <meta property="og:locale" content="en_GB" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="" />
```

Comment:
@ttbarnes @Zainzzkk need to talk about this as I am unable to fetch `pageTitle` value.

Reply:
Will have a look into this. I don't think it's possible to consume variables from a controller on this level. In other pages we use `CONTENT_STRINGS.PAGE_TITLE`, generated from core page variables and then call `{% block pageTitle %}`